src/main/java/it/gov/pagopa/payhub/activities/dto/paymentsreporting/PaymentsReportingDTO.java

```diff
@@ -0,0 +1,46 @@
+package it.gov.pagopa.payhub.activities.dto.paymentsreporting;
+
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+import java.util.Date;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentsReportingDTO {
+
+	private int version;
+	private Timestamp creationDate;
+	private Timestamp lastChangeDate;
+	private OrganizationDTO orgId;
+	private IngestionFlowFileDTO ingestionFlowFile;
```

Comment:
let's just store the id, not the relation if not needed to retrieve together with the entity
```suggestion
	private Long ingestionFlowFileId;
```